scripts/registerScript.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
Having `Script` in the filename for `registerScript.sh` `eraseScript.sh` seems redundant since we already know its a `.sh` file, no?  Up to you.